network/monuments.csv

```diff
@@ -233,6 +233,7 @@ WGTT,50208S004,Forced Centering,Steel Mast,-999,Stainless Steel Rod,0,1999-12-14
 WHKT,50244M001,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.5,Stainless Steel Rods,10,2009-03-23T00:00:00Z,9999-01-01T00:00:00Z,,
 WHNG,50218M001,Forced Centering,Pillar,-1.2,Reinforced Concrete,2,2003-04-09T00:00:00Z,9999-01-01T00:00:00Z,Limestone,
 WHVR,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.535,Stainless Steel Rods,10,2011-09-30T00:00:00Z,9999-01-01T00:00:00Z,,
+WIAH,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.5,Stainless Steel Rods,10,2024-10-09T00:00:00Z,9999-01-01T00:00:00Z,,
```

Comment:
Based on this photo from the site (20241009_021252762_iOS.jpg), I would say the ground relationship is -1.7 or -1.8. What do you think @EvanCGNS ?

Reply:
I think you're right, since he's standing slightly uphill from the monument I'll make it -1.8